CONTRIBUTING.md

```diff
@@ -0,0 +1,118 @@
+# Contributing
+
+- [Contributing](#contributing)
+  - [Contributing Code](#contributing-code)
+    - [Build Dependencies](#build-dependencies)
+    - [Building the SDK](#building-the-sdk)
+    - [Running Tests](#running-tests)
+    - [Git Commit Guidelines](#git-commit-guidelines)
+      - [Commit Message Format](#commit-message-format)
+      - [Revert](#revert)
+      - [Type](#type)
+      - [Scope](#scope)
+      - [Subject](#subject)
+      - [Body](#body)
+      - [Footer](#footer)
+    - [Submitting a Pull Request](#submitting-a-pull-request)
+  - [Updating the Documentation](#updating-the-documentation)
+    - [Set Up Environment (with Bundler)](#set-up-environment-with-bundler)
+    - [Compile and Serve Docs](#compile-and-serve-docs)
+
+## Contributing Code
+
+### Build Dependencies
+
+Before you can build the Cisco Webex Java SDK, you will need the following dependencies:
+
+- Java 1.6
+- [Apache Maven](https://maven.apache.org/)
+- [Git](https://git-scm.com/)
+
+### Building the SDK
+
+Fork the [webex-java-sdk](https://github.com/webex/webex-java-sdk) repository and `git clone` your fork:
+
+```bash
+git clone https://github.com/webex/your-username/webex-java-sdk.git
```

Comment:
```suggestion
git clone https://github.com/webex/webex-java-sdk.git
```

Reply:
I think this is correct, as the user will be cloning the forked repo. I was required to utilize the following command when developing against my forked repo / branch:
```
git clone https://github.com/InteractiveTimmy/spark-java-sdk.git
```

Reply:
I deleted the wrong part, sorry. It should be `https://github.com/your-username/webex-java-sdk`